code/datums/uplink/implants.dm

```diff
@@ -60,3 +60,10 @@
 	bluecrystal_cost = 5
 	desc = "A augment implanter, with a Hivenet defense augment and electronic warfare suite. ONLY WORKS ON VAURCA!"
 	path = /obj/item/device/augment_implanter/hivenet_warfare
+
+/datum/uplink_item/item/implants/telefreedom_kit
+	name = "Telefreedom Kit"
+	bluecrystal_cost = 15
+	item_limit = 1
+	desc = "A box containing a telefreedom full kit, with everything you need for 4 people. You supply the screwdriver and have to build it."
```

Comment:
this should explain what is a "telefreedom kit"

Reply:
cause someone who had never used it, would have no idea what it is and what does it do, just by looking at this text in the uplink program